plugins/topology/README.md

```diff
@@ -207,6 +207,27 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
     backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
   ```
 
+  If you have multiple entities while Red Hat Dev Spaces is configured and want multiple entities to be able to support the edit code decorator that redirects to the Red Hat Dev Spaces instance. You can add the `backstage.io/kubernetes-label-selector` annotation to the `catalog-info.yaml` file for each entity.
+
+  ```yaml title="catalog-info.yaml"
+  annotations:
+    backstage.io/kubernetes-label-selector: 'component in (<BACKSTAGE_ENTITY_NAME>,che)'
+  ```
+
+  If you are using the previous label selector, ensure that you add the following labels to your resources so that the Kubernetes plugin gets the Kubernetes resources from the requested entity:
+
+  ```yaml
+  labels: component=<BACKSTAGE_ENTITY_NAME>
+  ```
+
+  ```yaml title="devspaces.yaml"
```

Comment:
@debsmita1 to be more precise should we change this to `checluster.yaml` based on its kind?